docs/references/strategies/README.md

```diff
@@ -105,3 +104,39 @@
 - [carthage](platforms/ios/carthage.md)
 - [cocoapods](platforms/ios/cocoapods.md)
 - [swiftPM](platforms/ios/swift.md)
+
+## Strategies by type
+
+Languages supported by FOSSA CLI can have multiple strategies for detecting dependencies, one primary strategy that yields ideal results and zero or more fallback strategies. Within this list of strategies, we have the concept of _static_ and _dynamic_ strategies. Static strategies parse files to find a dependency graph (example: parse a `package-lock.json` file). Dynamic strategies are required when analyzing package managers that do not offer complete lockfiles, such as Gradle or Golang. Dynamic strategies require a working build environment to operate in.
+
+It is important to note that neither type of strategy has an inherent benefit when detecting dependencies. If a supported language does not have static and dynamic strategies, this does not mean it is less supported than a language that does.
```

Comment:
```suggestion
It is important to note that neither type of strategy has an inherent benefit when detecting dependencies. If a supported language has only a static or only a dynamic strategy, this does not mean it is less supported than a language that
```

Reply:
This suggestion reads a bit more clearly to me because I could read the earlier version as saying a language that has neither a static nor a dynamic strategy might be well supported.